ios/Classes/BetterPlayer.m

```diff
@@ -663,12 +665,45 @@ - (void)usePlayerLayer: (CGRect) frame
     }
 }
 
+- (void)preparePictureInPicture: (CGRect) frame
```

Comment:
memo:
Copy from `usePlayerLayer` and modify.

(`usePlayerLayer` is never used. Because we don't call `enablePictureInPicture`.)